tips/TIP-0026/indexer-rest-api.yaml

```diff
@@ -0,0 +1,854 @@
+openapi: 3.0.3
+info:
+  title: IOTA UTXO Indexer REST API
+  description: This document specifies the REST API for IOTA UTXO indexers.
+  contact:
+    email: contact@iota.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  version: 1.0.0
+externalDocs:
+  description: Find out more about IOTA
+  url: 'https://iota.org'
+servers:
+  - url: 'http://127.0.0.1:14265'
+tags:
+  - name: basic outputs
+    description: Query Basic Outputs.
+  - name: aliases
+    description: Query Alias Outputs.
+  - name: foundries
+    description: Query Foundry Outputs.
+  - name: nfts
+    description: Query NFT Outputs.
+paths:
+  /api/plugins/indexer/v1/basic-outputs:
```

Comment:
Is this to avoid a route called `/basics` ? Because it's a bit inconsistent with the other 3 output types :(

Reply:
If so, I'd rather have `/alias-outputs`, `/nft-outputs` and `/foundry-outputs` as well